src/settings.c

```diff
@@ -106,12 +106,20 @@ static settings_log_t client_log = NULL;
 /* Workaround for Cython not properly supporting variadic arguments */
 __attribute__((format(printf, 2, 3))) static void log_preformat(int level, const char *fmt, ...)
 {
-  char buffer[256];
+  char buffer[1024];
```

Comment:
it seems like we need to limit this to the buffer size allowed by SBP so we don't run into any serialization issues in FW